go/private/common.bzl

```diff
@@ -123,15 +123,27 @@ def has_simple_shared_lib_extension(path):
     """
     return any([path.endswith(ext) for ext in SHARED_LIB_EXTENSIONS])
 
+def is_valid_shared_lib_version_part(part):
+    """
+    Checks if the version part is valid. Matches the same check from
+    //src/main/java/com/google/devtools/build/lib/rules/cpp:CppFileTypes.java
+    """
+    if not part[:1].isdigit():
```

Comment:
```suggestion
    if not part[0].isdigit():
```
Simpler if it works

Reply:
It will fail on empty strings, but I'll check if it's possible to get an empty string here.

Reply:
I see, then let's keep this as is to not risk a "crash"